frontend/src/main/web/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Router from 'react-router';
+import RootContent from './lib/components/RootContent';
+import Views from './lib/constants/Views.js';
+import Configs from './lib/constants/Configs';
+import StringUtils from './lib/utils/StringUtils';
+import _ from 'lodash';
+import css from './index.css'
+
+var mountNode = document.getElementById('main-content'),
+  baseUrl = mountNode.getAttribute('base-url'),
+  user = JSON.parse(mountNode.getAttribute('user')),
+  data = JSON.parse(mountNode.getAttribute('data')),
+  view = Views.getView(data.view),
+  dev = data.dev;
```

Comment:
I am working on getting the config and setup instructions or script for webstorm, but I think it is better not to let it slow down this PR. We can keep this as-is for now, and sort out formatting for all the frontend code later (after it is merged).